packages/node/src/node.ts

```diff
@@ -303,6 +336,38 @@ export class Node {
     return this.requestHandler.callMethod(method, req);
   }
 
+  /**
+   * Registers a plugin for a given App
+   */
+  registerPlugin(plugin: Plugin, appId: string) {
+    // TODO: plugins should be mapped to appInstances?
+    this.plugins.set(appId, plugin);
+  }
+
+  /**
+   *
+   * @param appId
```

Comment:
we need to fix https://github.com/counterfactual/monorepo/issues/1377 instead of writing new code that continues to use this terminology

Reply:
good call